.flake8

```diff
@@ -1,248 +1,46 @@
 [flake8]
 ignore =
-    E20,   # Extra space in brackets
-    E231,E241,  # Multiple spaces around ","
-    E26,   # Comments
-    E731,  # Assigning lambda expression
-    E741,  # Ambiguous variable names
-    W503,  # line break before binary operator
-    W504,  # line break after binary operator
+    # Extra space in brackets
+    E20,
+    # Multiple spaces around ","
+    E231,
+    E241,
+    # Comments
+    E26,
+    # Assigning lambda expression
+    E731,
+    # Ambiguous variable names
+    E741,
+    # line break before binary operator
+    W503,
+    # line break after binary operator
+    W504,
+    E1,
```

Comment:
Does the addition of E1 and the following lines mean that it will now be possible to introduce violations into the CUDA target files (`numba/cuda`)? They were not part of the exclude list (I fixed all violations in the CUDA target a couple of years ago and removed them from the exclude list), but will this these ignores will now apply to them?

Reply:
Yes the ignore list was increased as part of `Temporarily ignore all flake8 codes` so that pre-commit-ci could run to autofix things. I could add follow up commits that do revert both `Temporarily ignore all flake8 codes` and `Remove grandfathered flake8 file exceptions` to keep the current state. I'm in a tough situation where I can't apply all bulk changes at once (see #8347), but to iterate in tiny bits I need to ignore more and then remove them again in subsequent PRs.